python/paddle/distributed/collective.py

```diff
@@ -984,7 +981,7 @@ def all_gather(tensor_list, tensor, group=None, use_calc_stream=True):
     """
 
     Gather tensors from all participators and all get the result. As shown
-    below, 4 GPUs each start 4 processes and the data on each GPU is represnted
+    below, 4 GPUs each starts 4 processes and the data on each GPU is represented
```

Comment:
One process is started with a GPU and the data of this process is represented by the group rank.

Reply:
Revised.